Template.ProcessCube.Project/README.md

```diff
@@ -1 +1,66 @@
-# # Template.ProcessCube.Project
+# Template.ProcessCube.Project
+Nach dem Erzeugen eines neuen Templates, kann dieses Bearbeitet werden
+## Grundvorraussetzungen
+
+### Docker
+
+<https://www.docker.com/get-started>
+
+### .NET 6.0
+
+<https://dotnet.microsoft.com/en-us/download/dotnet/6.0>
+
+### Node.js
+
+Um das Projekt starten zu können ist die Installation von Node.js vorrausgesetzt, welches von [der offiziellen Webseite](https://nodejs.org/en/download/)
+heruntergeladen werden kann.
+
+## Build und Start
+Dieser Abschnitt zeigt die Grundlegenden Befehle, die zum Starten des Projekts notwendig sind:
+
+### Basic Build
+Wenn nur ein csproj in der Solution enthalten ist, dann kann Projekt innerhalb der Solution gebaut werden mit
+
+```zsh
+dotnet build
+```
+, denn es wird dann das einzige Projekt in der Solution gebaut.
+
+Bei mehreren Projekten gehe in den Ordner apps/Template.ProcessCube.Project/
+```zsh
+cd apps/Template.ProcessCube.Project/
+```
```

Comment:
Du kannst auf mehrere Projekte auf einmal bauen, sofern sie der Solution hinzugefügt wurden. Da in den angegebenen Befehlen aber sowieso vorher ein `dotnet build` ausgeführt wird, ist der Punkt eh mehr oder minder obsolet